### ISSUE TITLE
Customer Import: update to `disable_auto_group_change` ignored

### ISSUE THREAD

**Issue by nbennett25** [magento/magento2#36409]
### Preconditions and environment

1.Magento 2.3.5-p2
2.Customer exists

### Steps to reproduce

1. Go to System > Import/Export > Export
2. Select type: Customer Main File
3. Export Customer record
4. In downloaded file, strip out all non-required fields except for `disable_auto_group_change`
5. Change `disable_auto_group_change` value(if the customer already has the disable auto group change checked in their customer record, switch the 1 above to 0)
6. Go to System > Import/Export > Import
7. Select Type: Customer Main File
8. Select Import Behavior: Add/Update Complex Data
9. Upload File
10. Check Data
11. Import File

### Expected result

1. Customer record shows change in 'Disable Automatic Group Change Based on VAT ID' setting.

### Actual result

1. No changes in 'Disable Automatic Group Change Based on VAT ID' setting.

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [X] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by nbennett25** [comment]
@magento give me 2.4-develop instance

**Comment by nbennett25** [comment]
Confirmed in `2.4-develop`

**Comment by engcom-Bravo** [comment]
@magento give me 2.4-develop instance

**Comment by engcom-Bravo** [comment]
Hi @nbennett25,

Thank you for reporting and collaboration.
Verified the issue on Magento 2.4-develop instance we are not able to reproduce the issue.Kindly Provide more information about disable_auto_group_change value and also provide exact steps to reproduce the issue and screenshots if required.


Kindly provide your CSV file it is helpful for us.


Thanks.

**Comment by nbennett25** [comment]
@engcom-Bravo I was able to reproduce this on the 2.4-develop instance provided. The csv content will be determined by the customer records (specifically email address), but the base of it is below:

```
_website,email,disable_auto_group_change
base,[customer email],1

```

**Comment by engcom-Bravo** [comment]
Hi @nbennett25,

Thanks for your update.

We tired to reproduce the issue but we are not able to reproduce the issue.we Followed these steps

1.Go to System > Import/Export > Export
2.Select type: Customer Main File
3.Export Customer record

We got Customer File :

[export_customer_20221110_010422.csv](https://github.com/magento/magento2/files/9979337/export_customer_20221110_010422.csv)


Kindly Provide more information about In downloaded file, strip out all non-required fields except for `disable_auto_group_change`,Change `disable_auto_group_change` value. It will helpful for us to proceed further.


Thanks.

**Comment by nbennett25** [comment]
@engcom-Bravo is there a way to attach files to a comment..? Please try creating a csv with the following (based on your file):

```
email,_website,_store,disable_auto_group_change
roni_cost@example.com,base,default,1
```
Then Import the file via System > Import/Export > Import, Customer Main File. (if the customer already has the disable auto group change checked in their customer record, switch the `1` above to `0` to flag the change.

**Comment by engcom-Bravo** [comment]
@magento give me 2.4-develop instance

**Comment by engcom-Bravo** [comment]
Hi @nbennett25,

Thanks for your Update.

You can attach files through a comment.

<img width="504" alt="Screenshot 2022-11-11 at 5 11 38 PM" src="https://user-images.githubusercontent.com/51680745/201333446-22ab0af6-539e-4ba1-aa7b-cadf4d65448a.png">


Kindly attach.

Thanks.

**Comment by nbennett25** [comment]
Here's a csv version of the above @engcom-Bravo:

[auto_group_change.csv](https://github.com/magento/magento2/files/9991379/auto_group_change.csv)

**Comment by engcom-Bravo** [comment]
@magento give me 2.4-develop instance

**Comment by engcom-Bravo** [comment]
Hi @nbennett25,

Thanks for your update.

Verified the issue on Magento 2.4-develop instance and the issue is reproducible.

Customer Import: update to `disable_auto_group_change` was ignored.


**Steps to reproduce**

1.Go to System > Import/Export > Export
2.Select type: Customer Main File
3.Export Customer record
4.In downloaded file, strip out all non-required fields except for `disable_auto_group_change`
5.Change `disable_auto_group_change value`(if the customer already has the disable auto group change checked in their customer record, switch the 1 above to 0)
6.Go to System > Import/Export > Import
7.Select Type: Customer Main File
8.Select Import Behavior: Add/Update Complex Data
9.Upload File
10.Check Data
11.Import File


We have used Below CSV file (existing customers) we changed `disable_auto_group_change` to `1` there is  No changes in 'Disable Automatic Group Change Based on VAT ID'



[five.csv](https://github.com/magento/magento2/files/10021068/five.csv)



Hence confirming this issue.

Thanks.
